### ISSUE TITLE
403 (Forbidden) for S3 resources

### ISSUE THREAD

**Issue by peterdesmet** [inbo/vespa-watch#311]
In Chrome I encountered the resources of the website not being loaded properly:

```
GET https://lw-vespawatch-prd.s3.amazonaws.com/static/vespawatch/libraries/leaflet/leaflet.css net::ERR_ABORTED 403 (Forbidden)
(index):31 GET https://lw-vespawatch-prd.s3.amazonaws.com/static/vespawatch/libraries/vue/vue-datetime.min.css net::ERR_ABORTED 403 (Forbidden)
(index):32 GET https://lw-vespawatch-prd.s3.amazonaws.com/static/vespawatch/css/main.css net::ERR_ABORTED 403 (Forbidden)
(index):254 GET https://lw-vespawatch-prd.s3.amazonaws.com/static/vespawatch/libraries/bootstrap/popper.min.js net::ERR_ABORTED 403 (Forbidden)
(index):255 GET https://lw-vespawatch-prd.s3.amazonaws.com/static/vespawatch/libraries/bootstrap/bootstrap.min.js net::ERR_ABORTED 403 (Forbidden)
(index):38 GET https://lw-vespawatch-prd.s3.amazonaws.com/static/vespawatch/img/avatar.jpg 403 (Forbidden)
```

I didn't encounter it on Safari over the same network. Neither did it occur for `uat.vespawatch.be`.

@niconoe any idea what might be wrong?

**Comment by niconoe** [comment]
No idea, I also cannot access those files and that breaks the website. The URLs looks correct and the website code hasn't changed in days. It really looks like some AWS (credentials) configuration issue. Can you get a hand on an AWS guru? @berthuygens maybe?

**Comment by niconoe** [comment]
This is broken on prod from everywhere to me (albeit some browsers may have those files in cache, thus keeping the website appears ok)... I initially had this on my phone.

The URLs also seems correct, it's just that S3 refuse to serve them. It really looks like some AWS configuration issue. Can @berthuygens or another AWS guru have a look?

**Comment by peterdesmet** [comment]
Same happens with dropzone thumbnails in UAT.

**Comment by niconoe** [comment]
This is pretty serious. @bartaelterman, it would be great if you could also have a look and confirm my impression it's AWS-related. I Feel like I only have half a brain today and don't want to blame others for issues in our code :)

**Comment by bartaelterman** [comment]
The url that is being requested is this: `https://lw-vespawatch-uat.s3.amazonaws.com/static/https://lw-vespawatch-uat.s3.amazonaws.com/media/`

The dropzone requests the url of the image via the API (at /api/nest_pictures). Locally, that url only returned the path (so without the domain name) and so I concatenated the static url with this path to get the full image url. On UAT it's apparently configured differently and the API already returns the full url, making this concatenation useless. I added code to check whether the schema is already in the string returned from the API and do the concatenation based on that check. 🤞
